### PR TITLE
[Feature] Add ability to import/export datahub configurations

### LINKED ISSUE #544: [Feature] Import/Export datahub configurations
### Feature description

As a user, I want to be able to export (download) existing endpoints/configurations - as with DataObject Classes. As the next step, I want to import those configurations as a new Datahub endpoint to a different or the same instance.
This should be possible for all types of datahub endpoints such as Simple Rest API, Graph QL API, File Export, DataObjectsImporter, Productsup, and CI-Hub.

### PR DESCRIPTION
**Pull request by Corepex**
resolves #544

### CODE REVIEW

File: src/Configuration.php [hunk 1]
```diff
@@ -53,6 +53,11 @@ class Configuration extends AbstractModel
      */
     public $group;
 
+    /**
+     * @var string
```
Comment (blankse) [inline src/Configuration.php]: Why not set typehints instead of php docs for new attributes/params?
```suggestion
     * @var string|null
```

Comment (Corepex) [reply src/Configuration.php]: I already changed the phpdoc annotations to typehints

File: src/Service/ImportService.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            ));
+        }
+
+        $configuration = Configuration::getByName($configuration['name']);
+        if ($configuration instanceof Configuration) {
+            throw new \Exception('Name already exists.');
+        }
+    }
+
+    protected function isBundleInstalled($namespace): bool
```
Comment (blankse) [inline src/Service/ImportService.php]:
```suggestion
    protected function isBundleInstalled(?string $namespace): bool
```

File: src/Controller/ConfigController.php [hunk 3]
```diff
@@ -555,4 +559,94 @@ public function getPermissionUsersAction(Request $request)
...
+            'name' => $configuration->getName()
+        ]);
+        // set content-type to text/html, otherwise (when application/json is sent) chrome will complain in
+        // Ext.form.Action.Submit and mark the submission as failed
+        $response->headers->set('Content-Type', 'text/html');
+
+        return $response;
+    }
+
+    private function getAllowedVars()
```
Comment (blankse) [inline src/Controller/ConfigController.php]:
```suggestion
    private function getAllowedVars(): array
```

File: src/Controller/ConfigController.php [hunk 4]
```diff
@@ -555,4 +559,94 @@ public function getPermissionUsersAction(Request $request)
 
         return $this->adminJson($users);
     }
+
+    /**
+     * @Route("/export", methods={"GET"})
+     *
+     * @param Request $request
+     */
+    public function exportConfiguration(Request $request, ExportService $exportService)
```
Comment (blankse) [inline src/Controller/ConfigController.php]:
```suggestion
     */
    public function exportConfiguration(Request $request, ExportService $exportService): Response
```

File: src/Controller/ConfigController.php [hunk 5]
```diff
@@ -555,4 +559,94 @@ public function getPermissionUsersAction(Request $request)
...
+
+    /**
+     * @Route("/import", methods={"POST"})
+     *
+     * TODO: add name as required param
+     *
+     * @param Request $request
+     * @param ImportService $importService
+     */
+    public function importConfiguration(Request $request, ImportService $importService)
```
Comment (blankse) [inline src/Controller/ConfigController.php]:
```suggestion
     */
    public function importConfiguration(Request $request, ImportService $importService): JsonResponse
```

**Pull request merged**